.chglog/config.yml

```diff
@@ -0,0 +1,29 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: CHANGELOG
+  repository_url: https://github.com/lamassuiot/lamassuiot
+options:
+  commits:
+    filters:
```

Comment:
I propose to adhere to a commit convention like this https://www.conventionalcommits.org/en/v1.0.0/#summary

Also, we should add a CONTRIBUTING.md file refering to these conventions.